Assets/MixedRealityToolkit/_Core/Devices/VoiceInput/DictationInputDeviceManager.cs

```diff
@@ -1,70 +1,33 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Core.Definitions.InputSystem;
+using Microsoft.MixedReality.Toolkit.Core.Interfaces.Devices;
 using Microsoft.MixedReality.Toolkit.Core.Interfaces.InputSystem;
+using Microsoft.MixedReality.Toolkit.Core.Managers;
 using Microsoft.MixedReality.Toolkit.Core.Utilities.Async;
 using Microsoft.MixedReality.Toolkit.Core.Utilities.Async.AwaitYieldInstructions;
+using System.Text;
 using System.Threading.Tasks;
 using UnityEngine;
 
 #if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
-using System.Text;
 using UnityEngine.Windows.Speech;
-#endif
+#endif // UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
 
-namespace Microsoft.MixedReality.Toolkit.InputSystem.Sources
+namespace Microsoft.MixedReality.Toolkit.Core.Devices.VoiceInput
 {
-    /// <summary>
-    /// Singleton class that implements the DictationRecognizer to convert the user's speech to text.
-    /// The DictationRecognizer exposes dictation functionality and supports registering and listening for hypothesis and phrase completed events.
-    /// </summary>
-    public class DictationInputSource : BaseGenericInputSource
+    // TODO - Implement
+    public class DictationInputDeviceManager : BaseDeviceManager, IMixedRealityDictationSystem
```

Comment:
I'd almost rather have a dedicated `WindowsDictationInputDeviceManager` and make each dictation manager be the platform specific implementation just like we did with the Gamepad Input.